src/winzone/WindowsTimeZoneIDs.jl

```diff
@@ -34,30 +35,67 @@ function compile(xml_file::AbstractString, translation_file::AbstractString)
         end
     end
 
+    return translation
+end
+
+function compile(xml_file::AbstractString, translation_file::AbstractString)
     open(translation_file, "w") do fp
-        serialize(fp, translation)
+        serialize(fp, compile(xml_file))
```

Comment:
what's the motivation for using the Julia serializer here? that isn't really meant for long-term storage